.github/workflows/main-branch-notifications.yml

```
@@ -20,5 +20,5 @@ jobs:
             📝 **Commit to Main Branch**
             **Commit**: ```${{ github.event.head_commit.message }}```
             **SHA**: `${{ github.sha }}`
-            **Author**: ${{ github.actor }}
+            **Author**: ${{ github.event.head_commit.author.username || github.event.head_commit.author.name || github.event.sender.login || github.actor }}
             **[View Commit](${{ github.event.head_commit.url }})**
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)